app/src/components/fields/searchField/searchField.scss

```diff
@@ -17,3 +17,7 @@
 .search-field {
   overflow: hidden;
 }
+
+.search-field--transparent {
+  background: transparent !important;
```

Comment:
What style are we trying to override with important here?

Reply:
white background for search component

Reply:
@KrDior I suppose this already fixed on [ui-kit side](https://github.com/reportportal/ui-kit/pull/90), pls make sure to use the latest version

Reply:
removed